mlflow/store/model_registry/abstract_store.py

```diff
@@ -442,6 +442,8 @@ def _await_model_version_creation_impl(self, mv, await_creation_for, hint=""):
                     f".{hint}"
                 )
             mv = self.get_model_version(mv.name, mv.version)
+            if mv.status != pending_status:
+                break
```

Comment:
@itepifanio I think this is simpler than assignment expression.

Reply:
Sure, agreed!